whitelist.txt

```diff
@@ -22,12 +22,16 @@ apple.co
 #Whatsapp
 wa.me
 # Misc shortners
+smarturl.it
+lnk.to
+hyperurl.co
 tny.im
 gg.gg
 cutt.ly
 # Misc
 i.ibb.co
 apkcombo.com
+wowhead.com
```

Comment:
Needs to be es.wowhead.com; For some reason only that one is blocked in Phishing.Database